Integrations/FeedOffice365/FeedOffice365_test.py

```diff
@@ -19,7 +19,7 @@ def test_fetch_indicators_command():
 
 @pytest.mark.parametrize('command, args, response, length', [
     (get_indicators_command, {'limit': 2, 'indicator_type': 'IPS'}, RESPOSNE_DATA, 4),
-    (get_indicators_command, {'limit': 2, 'indicator_type': 'URLS'}, RESPOSNE_DATA, 5)
+    (get_indicators_command, {'limit': 2, 'indicator_type': 'URLS'}, RESPOSNE_DATA, 6)
```

Comment:
Can we please change this variable name to `RESPONSE_DATA` both here everywhere it appears in your unit-test code and also in `test_data/feed_data.py`?